src/legacy/core_plugins/visualizations/public/types/types_service.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// @ts-ignore
+import { defaultFeedbackMessage } from 'ui/vis/default_feedback_message';
+// @ts-ignore
+import { VisProvider as Vis } from 'ui/vis/index.js';
+// @ts-ignore
+import { VisFactoryProvider as VisFactory } from 'ui/vis/vis_factory';
```

Comment:
@ppisljar Based on your comments I renamed `VisProvider -> Vis` and `VisFactoryProvider -> VisFactory` here. When I do a pass to update #35764 with the new imports, I can also rename any instances where they are used.

If you think it will be confusing to temporarily have something exposed as `Vis` even though it is consumed as a provider, we can revert back; I don't have strong feelings on this either way.